src/routes/index.svelte

```diff
@@ -23,6 +23,18 @@
 
     const LINE_COLORS = ['#b52b26', '#1c2db0', '#0f9429', '#d17819']
 
+    let priceSelected = 'close'
+    const candleAttributes = [
```

Comment:
Unclear what this variable is currently. Can we change it to a constant format like`PRICE_OPTIONS`?